### DISCUSSION TITLE
Is there any configuration that can ignore some specified classes

### DISCUSSION THREAD

**Discussion by 807801002** [apache/skywalking#9026] [Q&A]
I want to ignore some span from some specified classes，not ignore the whole trace.

**Comment by wu-sheng** [comment]
It depends on which plugins you are using. We don't instrument specified classes unless they are relative to RPC in default.

**Reply by 807801002** [reply]
It is the spring-annotation-plugin，I want to collect spring annotation span, but I want to ignore some specified classes in my segment.

**Reply by wu-sheng** [reply]
I am not an agent user, but I think you can't.
